plugins/katello/3.3/release_notes/release_notes.md

```diff
@@ -4,16 +4,42 @@ title: Release Notes
 version: 3.3
 ---
 
-# Katello Nightly () Release Notes
+# Katello 3.3 (Baltic Porter) Release Notes
 
-{% if page.version and page.version != '3.3' %}
 For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/KATELLO-{{ page.version }}/CHANGELOG.md).
-{% else %}
-For the full release notes, see the [Changelog](https://github.com/Katello/katello/blob/master/CHANGELOG.md).
-{% endif %}
 
 ## Features
 
+### Updated UI
+
+### Improved Manifest Import and Delete
+
+Manifest import and delete actions were refactored for Katello 3.3 in anticipation of Candlepin 2.0. Now users are able to see a step-by-step breakdown of tasks for these actions in the Dynflow console.
+
+### Username and Password for Repositories
+
```

Comment:
Can set the username/password to access upstream yum/puppet repositories or docker registries that require http basic authentication.